pkg/syncer/syncer.go

```diff
@@ -14,11 +14,19 @@ import (
 	"github.com/bitnami-labs/charts-syncer/pkg/utils"
 )
 
+// Clients holds the source and target chart repo clients
+type Clients struct {
+	src core.ClientV2
+	dst core.ClientV2
```

Comment:
I would try to avoid the word `destination` in favor of `target`, to be consistent with the config file and other variables/functions

Reply:
can we refactor this wording after I merge all the PRs? Otherwise, I need to rebase and fix the #57, #58 and #59

Reply:
makes sense yes!